cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -193,9 +193,14 @@
     # NOTE: at this stage the /snap directory is stable as we have called
     # pivot_root already.
 
-    # Support mount profiles via the content interface
-    mount options=(rw bind) /snap/*/** -> /snap/*/*/**,
+    # Support mount profiles via the content interface. This should correspond
+    # to permutations of $SNAP, $SNAP_DATA and $SNAP_COMMON for both reading
+    # and writing except for cases where a writable location is bind mounted to
+    # read-only location ($SNAP).
     mount options=(ro bind) /snap/*/** -> /snap/*/*/**,
+    mount options=(ro bind) /snap/*/** -> /var/snap/*/*/**,
```

Comment:
The comment above doesn't describe this new rule (only the other two). Please adjust and LGTM